server/gotenberg/gotenberg_test.go

```diff
@@ -0,0 +1,62 @@
+package gotenberg
+
+import (
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestStatusRunning(t *testing.T) {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		require.Equal(t, "/ping", r.URL.Path)
+		require.Equal(t, http.MethodGet, r.Method)
+	}))
+	defer ts.Close()
+	gt := New(ts.URL)
+	isRunning, err := gt.Status()
+	require.NoError(t, err)
+	require.True(t, isRunning)
+}
+
+func TestStatusNotRunning(t *testing.T) {
+	gt := New("http://not-existent-host")
```

Comment:
Suggest not making a DNS request out to `not-existent-host` -- could we mock an `httptest` server that instead doesn't respond to the ping?

Reply:
Right, nice catch, thanks! I'll just change it to `http://localhost`.

It's probably not needed to use `httptest` here since in this test, we need to test the behavior when Gotenberg server is not running at all.

Reply:
solved by 68f3a3951a40b7bb7c75285dfbac2e0e2185e602